.github/workflows/ccm.yml

```diff
@@ -51,7 +52,7 @@ jobs:
     
   release: 
      # Making sure that release only runs for tag pushes
-    if: startsWith(github.ref, 'refs/tags/') && github.repository_owner == 'tl-its-umich-edu'
+    if: startsWith(github.ref, 'refs/tags/') && (github.repository_owner == 'tl-its-umich-edu' || github.repository_owner == 'jaydonkrooss')
     needs: build # This ensures the build job finishes successfully before starting this job
```

Comment:
This code should be removed now since it served it's purpose

Reply:
removed